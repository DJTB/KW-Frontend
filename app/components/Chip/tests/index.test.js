// import Chip from '../index';


// import { shallow } from 'enzyme';
// import React from 'react';

describe('<Chip />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
