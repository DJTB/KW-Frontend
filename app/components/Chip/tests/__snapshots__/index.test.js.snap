// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chip /> should match baseline snapshot 1`] = `
<styled.li
  bgColor="white"
  color="blackLight"
>
  <styled.span>
    text
  </styled.span>
</styled.li>
`;
